src/app/repositories/userProfile.repository.ts

```diff
@@ -0,0 +1,7 @@
+import { UserProfile } from "@models/entities";
+import { AppDataSource } from "@root/ormconfig";
+
+const userProfileRepository =
+  AppDataSource.getRepository(UserProfile);
+
+export { UserProfile, userProfileRepository };
```

Comment:
[R] Same concerns, remove `UserProfile` from export
```suggestion
export { userProfileRepository };
```